src/hud/SearchBox.ts

```diff
@@ -3,6 +3,12 @@ import { Viewer } from '@satellite-viewer/index';
 import SatelliteGroup from '@satellite-viewer/SatelliteGroup';
 import HudWindowManager from './HudWindowManager';
 
+interface SearchResults {
```

Comment:
Can you put this in `src/viewer/interfaces` as a standalone file?

Reply:
Done.